Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
@dsyme ...
Well I suppose I can do this in MSBuild ... so sure no problem ... it will be fun to try figuring out what is going on then :-)